.travis.yml

```diff
@@ -20,3 +20,15 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.0" STRICT_VARIABLES="yes"
 notifications:
   email: false
+
+deploy:
+  provider: puppetforge
+  user: puppet
+  password:
+    secure: "IkrfAnec7ovZLMvhvXt8ZihyYdAJTC/nm7KDm4u2G/uD2NGaMdHNOAenkwIwC1vfCzHKcgC5u/lAYFrYvHpQpJW0kHLKnk1SpndfWX9kd5SlDDzEP5mJGjMZeTY6H9sV5fsB6Pt7l/sw5ACL/0bFDl0mYBnVhGv6UxZZ5xMQIUw="
```

Comment:
This actually needs to be:

```
secure: "SCQpiBB9qpZAjBRk+b9D3cSCQfYpDgHPOdOc7djfGeB5yn1UbGg7uW1zshrshb4QMLfUgvsL4LsT0CYj7ilBerghgeySF5JWuZdk05W/7Iudls4btbxdVjqtALR7p02mnk40qHTR1Tdb/j0gXW9uigU6nQU9iCP+Poa1KF6PXpk="
```

This comes from https://github.com/puppet-community/plumbing/blob/master/share/travis_secrets